ggml/src/CMakeLists.txt

```diff
@@ -139,6 +139,19 @@ if (GGML_METAL)
         )
 endif()
 
+if (GGML_MUSA)
+    set(CMAKE_C_COMPILER clang)
+    set(CMAKE_C_EXTENSIONS OFF)
+    set(CMAKE_CXX_COMPILER clang++)
+    set(CMAKE_CXX_EXTENSIONS OFF)
+
+    set(GGML_CUDA ON)
+
+    list(APPEND GGML_CDEF_PUBLIC GGML_USE_MUSA)
+
+    add_compile_definitions(GGML_USE_MUSA)
```

Comment:
The `add_compile_definitions` is redundant here, adding to `GGML_CDEF_PUBLIC` is enough.